spec/controllers/challenge_claims_controller_spec.rb

```diff
@@ -4,8 +4,9 @@
 describe ChallengeClaimsController do
   include LoginMacros
   include RedirectExpectationHelper
+
   let(:user) { create(:user) }
-  let(:signup) { create(:challenge_signup) }
+  let(:signup) { create(:prompt_meme_signup) }
```

Comment:
Can we remove the same `let` in a nested context?

https://github.com/otwcode/otwarchive/blob/27d089d5189069609eeea552e6a7d2ba52c7917e/spec/controllers/challenge_claims_controller_spec.rb#L30